.buildkite/pipeline.yml

```diff
@@ -75,8 +77,10 @@ steps:
         --argstr commitSha "$(git rev-parse HEAD)"
         --argstr commitDate "$(git log HEAD -1 --format=%cd)"
     - ln -s ./result/BaseDAO.md BaseDAO.md
+    - ln -s ./result/GameDAO.md GameDAO.md
```

Comment:
Btw, looks like it is not too hard to do [a batch symlinks creation](https://superuser.com/a/633610/756653), maybe use it?

Reply:
Let's see what CI will tell us :zhdun: